src/sdk/evaluation.ts

```diff
@@ -0,0 +1,2 @@
+export type {default as Evaluator, EvaluationOptions} from '@croct/sdk/facade/evaluatorFacade';
+export {EvaluationError, EvaluationErrorType, ExpressionError} from '@croct/sdk/evaluator';
```

Comment:
The SDK is now a public package and a dependency of the plug, so everyone that installs the plug from npm will also install the SDK. These interfaces will be exported both from `@croct/plug/...` and `@croct/sdk/...`.

I'm not sure how intellisense and autocomplete will behave in the IDEs for those. Normally they use the smallest import path, but even if all of them do `@croct/plug/sdk` and `@croct/sdk/evaluator` is a tie. I'm worried that it will end in some weird mixed import.

Maybe we could just leave those types on the SDK and allow the language-server to find it out when someone needs them? I don't like the idea, but... well...

Reply:
Maybe direct dependencies have a higher priority than transitive ones. That would make this ok, but I'll have to check that

Reply:
Although it's feasible, I don't think it's a good idea because you should never rely on a transitive dependency (theoretically we could remove the SDK dependency). Making this move would require us to recommend also installing a peer dependency that is really annoying.

Reply:
Also, SDK is a heavy package with hundreds of classes that make discovering class harder. By reexporting only the plugin-related classes, we decrease the cognitive complexity and improve discoverability.

Reply:
Yeah, _apparently_ direct dependencies do have a higher priority, but I could not find any reference to this on their docs and I haven't explored the source code. I just tested the autocomplete on Atom's IDE and SublimeText using the official language-server and on WebStorm using their custom fork